src/type/preload.ts

```diff
@@ -285,12 +285,23 @@ export type DefaultStyleId = {
   defaultStyleId: number;
 };
 
-export type MinimumEngineManifest = {
-  name: string;
-  uuid: EngineId;
-  command: string;
-  port: string;
-};
+export const supportedFeaturesItemSchema = z.object({
+  type: z.string(),
+  value: z.boolean(),
+  name: z.string(),
+});
+
+export const minimumEngineManifestSchema = z
+  .object({
+    name: z.string(),
+    uuid: z.string(),
```

Comment:
あ、ここ型が変わってそうです！
```suggestion
    uuid: engineIdSchema,
```

ちょっとこちらで取り込んじゃってマージしたいと思います！！